src/plugins/data/common/search/aggs/buckets/range.ts

```diff
@@ -67,11 +68,11 @@ export const getRangeBucketAgg = ({ getFieldFormatsStart }: RangeBucketAggDepend
         keyCaches.set(agg, keys);
       }
 
-      const id = RangeKey.idBucket(bucket);
+      const id = RangeKey.idBucket(bucket, agg.params.ranges);
```

Comment:
Forgive me if this is a naive question -- but why is it necessary to change the format of the bucket id to include the label? I assume this is solving an edge case, but it wasn't completely clear to me from reading the discussion above.

Reply:
I was under the impression it's necessary to not run into cached `RangeKey` objects with outdated labels, but I realized the cache is only used for a single request, so it's not an issue. Removed it